tests/unit/types/document/test_converters.py

```diff
@@ -83,6 +84,18 @@ def test_buffer_to_blob():
     assert doc.blob.shape == (85, 152, 3)  # h,w,c
 
 
+def test_image_blob_to_file_with_buffer():
```

Comment:
please use `tmpdir` fixture if you plan to dump somewhere? where is this `file` found?

Reply:
Hey im not sure what you mean, im not dumping anything to a file?
Im just using an inmemory bytesio object.
If you mean the "test.png" image path, i copie that from the test 'test_buffer_to_blob' above

Reply:
fair enough